res/css/views/settings/tabs/_SettingsTab.scss

```diff
@@ -26,21 +26,19 @@ limitations under the License.
     font-size: $font-20px;
     font-weight: 600;
     color: $primary-content;
-    margin-bottom: 10px;
-    margin-top: 10px;
-}
+    margin-block: 10px; // TODO: Use a spacing variable
```

Comment:
I think we should stick here to `margin-bottom`/`margin-top`. If you want to use a shorthand property you can use `margin`. `margin-block` depends on some other attributes. Here the margins are used to define the layout which requires space above and below the element. `margin-block` [can be affected by other CSS properties](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block).

Applies to other places here as well.

Reply:
ok, not really a must unless we would enable UI in [vertical texts](https://www.w3.org/International/articles/vertical-text/) :-D